go.mod

```diff
@@ -1,11 +1,16 @@
 module github.com/Microsoft/go-winio
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.10
```

Comment:
You can probably remove this one if it automatically added it (go modules tends to randomly add these, but adding also means that users get either down- or upgraded when running commands, so I try to avoid adding these.

Reply:
Yes, I’ll remove the toolchain.